ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -533,6 +533,20 @@ interface TestController {
      * @param options - options.
      */
     request: Request;
+
+    /**
+     * Set skipJsErrors options
+     *
+     * @param options - options or boolean.
+     */
+    skipJsErrors (options?: boolean | SkipJsErrorsOptions): TestControllerPromise;
+
+    /**
+     * Set a handler function that will be called for each error.
+     *
+     * @param options - SkipJsErrorsCallbackOptions or SkipJsErrorsCallback.
+     */
+    skipJsErrors (options: SkipJsErrorsCallback | SkipJsErrorsCallbackOptions): TestControllerPromise;
```

Comment:
I see the final result and I think now, that it's better to combine all methods into one method.